src/promptflow/promptflow/_sdk/entities/_run.py

```diff
@@ -642,7 +642,7 @@ def _validate_and_return_run_name(run: Union[str, "Run"]) -> str:
     def _validate_for_run_create_operation(self):
         """Validate run object for create operation."""
         # check flow value
-        if Path(self.flow).is_dir():
+        if Path(self.flow).is_dir() or Path(self.flow).is_file():
```

Comment:
@0mza987 could you help to check if we can support file here?

Reply:
LGTM, but may need to refine the docstring of the Run object about this parameter.